part-spec/passives.json

```diff
@@ -0,0 +1,350 @@
+{
+    "$id": "https://github.com/chromeos/digital-datasheets/blob/main/part-spec/passives.json",
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "title": "passives",
+    "oneOf": [
+        {
+            "title": "resistor",
+            "type": "object",
+            "required": [
+                "partType",
+                "manufacturer",
+                "orderableMPN",
+                "sourceDatasheetID",
+                "digitalDatasheetID"
+            ],
+            "properties": {
+                "partType": {
+                    "description": "part type",
+                    "type": "string",
+                    "enum": [
+                        "resistor"
+                    ]
+                },
+                "manufacturer": {
+                    "description": "company that manufactures the part",
+                    "type": "string"
+                },
+                "componentName": {
+                    "description": "base part name that describes the form and fit of a component",
+                    "type": "string"
+                },
+                "orderableMPN": {
+                    "description": "orderable manufacturer part number",
+                    "type": "string"
+                },
+                "sourceDatasheetID": {
+                    "description": "methods for identifying the human-readable source information for a digital datasheet",
+                    "$ref": "#/$defs/sourceDatasheetID"
+                },
+                "digitalDatasheetID": {
+                    "description": "methods for identifying the version of the digital datasheet",
+                    "$ref": "#/$defs/digitalDatasheetID"
+                },
+                "status": {
+                    "description": "production status of a component",
+                    "examples": [
+                        "preProduction",
+                        "active",
+                        "EOL"
+                    ],
+                    "type": "string"
+                },
+                "pins": {
+                    "description": "array of pin objects with associated properties",
+                    "type": "array",
+                    "items": {
+                        "$ref": "definitions.json#/pinSpec"
```

Comment:
There's a discrepancy between the reference here and the one for the units. "definitions.json" vs "definitions". pinSpec & pins are in the same file. Let's check this